.harness/nodejsstarterpipeline.yaml

```diff
@@ -0,0 +1,96 @@
+pipeline:
+  name: nodejs-starter-pipeline
+  identifier: nodejsstarterpipeline
+  projectIdentifier: DhrubaCI
```

Comment:
Change the Project Identifier to a generic name

Reply:
removed the identifiers as they'll be unique to the user